README.md

```diff
@@ -61,6 +60,108 @@ yarn start
 
 Finally, we can navigate a browser window to `localhost:8080` to view generated data.
 
+### Running with Docker
+
+You can build and run sourcecred in a container to avoid installing dependencies
+on your host. First, build the container:
+
+```bash
+$ docker build -t sourcecred .
+```
+
+If you want to build and customize the `SOURCECRED_DIRECTORY`, you can
+set that as a `--build-arg`:
+
+```bash
+$ docker build --build-arg SOURCECRED_DEFAULT_DIRECTORY=/tmp/data -t sourcecred .
+```
+
+You will still need to export a GitHub token, and then provide it to the container
+when you run it. Notice that we are also binding port 8080 so we can
+view the web interface that will be opened up.  The only argument needed is a command to load the GitHub repository to generate the sourcecred for:
```

Comment:
Please wrap this text for consistency with the rest of the file

Reply:
You mean one sentence per line, with wrapping as necessary? I'll fix this now. @Beanow I'll also rename containers to sourcecred/sourcecred.

Reply:
Heh, I should stop bothering people about this and just find an automated solution to impose some consistent style. Do whatever seems best :)